testdata/station_data.txt

```diff
@@ -742,7 +742,7 @@ time,station,latitude[unit="degrees_north"],longitude[unit="degrees_east"],air_p
 2016-01-15 23:53:00Z,LXV,39.220,-106.319,NaN,-9.0,1.0,-13.0,NaN,NaN,0.75,1800.0,1.0,2100.0,NaN,NaN,10.0,,320,3.086664,
 2016-01-15 23:54:00Z,LYH,37.330,-79.199,NaN,4.0,1.0,4.0,NaN,NaN,1.0,120.0,NaN,NaN,NaN,NaN,5.0,-RA BR,360,3.086664,
 2016-01-16 00:15:00Z,LYV,43.619,-96.220,NaN,-10.0,1.0,-14.0,NaN,NaN,1.0,1950.0,NaN,NaN,NaN,NaN,10.0,,340,4.629996,AO2
-2016-01-15 23:45:00Z,LZU,33.979,-83.959,NaN,9.0,1.0,6.0,NaN,NaN,1.0,150.0,NaN,NaN,NaN,NaN,5.0,,250,4.115552,ATIS X, RWY 25 JW
+2016-01-15 23:45:00Z,LZU,33.979,-83.959,NaN,9.0,1.0,6.0,NaN,NaN,1.0,150.0,NaN,NaN,NaN,NaN,5.0,,250,4.115552,ATIS X RWY 25 JW
```

Comment:
What was going on here with the comma?

Reply:
TDS didn't properly escape the ',' in the remark column, so I'm just removing it.